.eslintrc.js

```diff
@@ -87,6 +87,7 @@ const INLINE_NON_VOID_ELEMENTS = [
         caseInsensitive: true,
       },
     }],
+    'no-useless-constructor': 'off',
```

Comment:
**Suggestion**: We shouldn't just turn off rules that are generally sensible. In this case, we should enable the TS-specific rule so the constructor in Api.ts is not considered useless.

```suggestion
    'no-useless-constructor': 'off',
    '@typescript-eslint/no-no-useless-constructor': 'error',
```

Reply:
I'm going to revert this, all the tests have blown up

Reply:
Well, but the error is correct: in JavaScript, this would be a useless constructor. However, in TypeScript, it's not. It's a little annoying that we need to manually turn off rules and enable their TS-counterpart like that though.